spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -704,6 +704,29 @@ function getCounterPartyHops(proof: CommitmentProof | MultihopProof, lastConnect
 }
 ```
 
+##### Helper functions
+
+```typescript
+// Returns the status of a client given its store.
+function Status (client: clientState) {
+  if (client.FrozenHeight !== 0) {
+    return Frozen
+  }
+  // Get latest consensus state from clientStore to check for expiry
+  consState, err := client.latestClientHeight()
+  if err (!== nil) {
+    return Unknown
+  }
+  // Check if Expired
+  let expirationTime := consState.Timestamp + client.TrustingPeriod
+  if (expirationTime <== now){
+    return Expired 
+  }
+
+  return Active
+} 
```

Comment:
Since this implementation of `Status` is specific to Tendermint, I think I would move it to ICS 07.